seed/serializers/properties.py

```diff
@@ -276,12 +293,20 @@ def to_representation(self, data):
             iterable = data
             view_ids = [view.id for view in iterable]
             results = []
+
+            # Grab extra_data columns to be shown in the result
+            organization = data[0].state.organization_id
```

Comment:
What is the expected value/instance type of `data` in this context?

Reply:
data is the django db object instance. `result` would be a dict.